hack/e2e/eksctl.sh

```diff
@@ -69,13 +70,19 @@ function eksctl_create_cluster() {
     ${BIN} create nodegroup \
       --managed=false \
       --cluster="${CLUSTER_NAME}" \
-      --node-ami-family=WindowsServer2019FullContainer \
+      --node-ami-family=WindowsServer2022FullContainer \
       -n ng-windows \
-      -m 1 \
-      -M 1 \
+      -m 3 \
+      -M 3 \
       --ssh-access \
       --ssh-public-key "${SSH_KEY_PATH}".pub
-    ${BIN} utils install-vpc-controllers --cluster="${CLUSTER_NAME}" --approve
+
+    cluster_role=$(aws eks describe-cluster --name ${CLUSTER_NAME} --query cluster.roleArn --output text | cut -d / -f 2)
```

Comment:
Is ``cluster_role`` used anywhere?

Reply:
We can remove this snippet altogether. We patch in the `VPCResourceControllerPolicy` already: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/d889329395deee4ed8fcccbd6bf4b204ac9d7f7d/hack/eksctl-patch.yaml#L2

https://github.com/weaveworks/eksctl/blob/5035eaf122d9ec0bedd15d1cde5247906acb41a9/pkg/apis/eksctl.io/v1alpha5/iam.go#L46